docs/source/markdown/podman-artifact-ls.1.md

```diff
@@ -0,0 +1,55 @@
+% podman-artifact-ls 1
+
+
+## WARNING: Experimental command
+*This command is considered experimental and still in development. Inputs, options, and outputs are all
+subject to change.*
+
+## NAME
+podman\-artifact\-ls - List OCI artifacts in local store
+
+## SYNOPSIS
+**podman artifact ls** [*options*]
+
+## DESCRIPTION
+
+List all local artifacts in your local store.
+
+## OPTIONS
+
+#### **--format**
+
+Print results with a Go template.
+
+| **Placeholder** | **Description**                                |
+|-----------------|------------------------------------------------|
+| .Digest         | The computed digest of the artifact's manifest |
+| .Repo           | Repository name of the artifact                |
```

Comment:
This (and in implementation) should be `.Repository`  to be consistent with podman images